packages/esm-appointments-app/src/appointment-forms/create-appointment-form.scss

```diff
@@ -0,0 +1,145 @@
+@import '~@openmrs/esm-styleguide/src/vars';
+@import '~carbon-components/src/globals/scss/vars';
+@import '~carbon-components/src/globals/scss/mixins';
+@import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/type/styles';
+@import '~carbon-components/src/globals/scss/vendor/@carbon/layout/scss/generated/spacing';
+
+// TO DO Move this styles to style - guide
+// https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-styleguide/src/_vars.scss
+$color-blue-30: #a6c8ff;
```

Comment:
`$carbon--blue-30` should exist in the above imports

Reply:
I thought so too but it wasn't there in the [var.scss](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-styleguide/src/_vars.scss)

Reply:
With the merge to 4.0 and carbon v11 it should exist in @carbon/colors. Let me double check the new naming for this color...